book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def rentals(person, date)
+    Rental.new(date, person, self)
```

Comment:
- [ ] Kindly arrange the parameters.

  ```
  Rental.new(date, self, person)
  ```

Reply:
they have been arranged, thanks for giving me such progressive review 💯